mirrord/cli/src/main.rs

```diff
@@ -102,6 +103,85 @@ where
     // Put original executable in argv[0] even if actually running patched version.
     binary_args.insert(0, args.binary.clone());
 
+    // Print config details for the user
+    sub_progress.info(
+        &format!(
+            "Running binary \"{}\" with arguments: {:?}.",
+            binary, args.binary_args
+        )[..],
+    );
+
+    let target_info = if let Some(target) = &config.target.path {
+        &format!("mirrord will target: {}", target)[..]
+    } else {
+        // TODO: check accuracy of statement
+        "mirrord will run without a target"
+    };
+    let config_info = if let Ok(path) = std::env::var("MIRRORD_CONFIG_FILE") {
+        &format!("a configuration file was loaded from: {}. ", path)[..]
+    } else {
+        "no configuration file was loaded"
+    };
+    sub_progress.info(&format!("{}, {}", target_info, config_info)[..]);
+
+    let operator_info = match config.operator {
+        Some(true) => "be used",
+        Some(false) => "not be used",
+        None => "be used if possible",
+    };
+    sub_progress.info(&format!("operator: the operator will {}", operator_info)[..]);
+
+    let exclude = config.feature.env.exclude.as_ref();
+    let include = config.feature.env.include.as_ref();
+    let env_info = if exclude.is_some() {
+        if exclude
+            .unwrap()
+            .clone()
+            .to_vec()
+            .contains(&String::from("*"))
+        {
+            "no"
+        } else {
+            "not all"
+        }
+    } else if include.is_some() {
+        "not all"
+    } else {
+        "all"
+    };
+    sub_progress.info(&format!("env: {} environment variables will be fetched", env_info)[..]);
+
+    let fs_info = match config.feature.fs.mode {
+        FsModeConfig::Read => "read only from the remote",
+        FsModeConfig::Write => "read and write from the remote",
+        _ => "read and write locally",
+    };
+    sub_progress.info(&format!("fs: file operations will default to {}", fs_info)[..]);
```

Comment:
maybe mention the defaults local/remote that are overriding (unless explicitly specified otherwise)

Reply:
Can you give an example? I don't think I can visualise it

Reply:
[here](https://github.com/metalbear-co/mirrord/tree/main/mirrord/layer/src/file/filter) we have default regex sets for file ops. There's quite a lot of them tbh, I don't know if we should list them all :eyes: 

I think the ideal solution would be to have them at `mirrord.dev` docs and only provide a link in this message (probably not in scope of this PR though). @aviramha wdyt?

Reply:
Yeah that's what I had in mind as well. Thanks @Razz4780